### ISSUE TITLE
Training loss

### ISSUE THREAD

**Issue by aigerimhatake** [maziarraissi/PINNs#30]
Hello!

How to plot training loss for Burgers equation in TF in order to understand if the model overfits or underfits?
